core/src/main/java/io/questdb/log/LogRollingFileWriter.java

```diff
@@ -51,20 +52,30 @@ public class LogRollingFileWriter extends SynchronizedJob implements Closeable,
     private long _wptr;
     private long buf;
     private String bufferSize;
+    private long currentLogSizeSum;
     private long currentSize;
     private int fd = -1;
     private long idleSpinCount = 0;
+    private String lifeDuration;
     private long lim;
     // can be set via reflection
     private String location;
+    private String logDir;
+    private NativeLPSZ logFileName = new NativeLPSZ();
```

Comment:
nit: should be `final`:
```suggestion
    private final NativeLPSZ logFileName = new NativeLPSZ();
```